src/caret_analyze/value_objects/variable_passing.py

```diff
@@ -93,7 +121,7 @@ def node_name(self) -> str:
         Returns
         -------
         str
-            node name
+            Node name
```

Comment:
Please append a period.

Reply:
Fixed it in [053d3c9](https://github.com/rokamu623/CARET_analyze/pull/23/commits/053d3c9bb8f0879da4c23ed92c13a96ef6728faf).